docs/content/about-iota/iota-tools/shimmer-bridge.md

```diff
@@ -0,0 +1,350 @@
+# ShimmerBridge User Guide
```

Comment:
Eahm, this should not be in the IOTA docs tbh.
We should not mention Shimmer anywhere in the IOTA docs.
If we need docs for Shimmer these should be separate and hosted under docs.shimmer.network since they will have completely different tech stacks.